helpers/opentracing.go

```diff
@@ -0,0 +1,34 @@
+package helpers
```

Comment:
this might belong in a different package? (or eventually not in this repo altogether)